monasca-log-agent/monasca-log-agent.conf.p2

```diff
@@ -0,0 +1,40 @@
+input {
+  udp {
+      port => {{ LOGSTASH_INPUT_PORT }}
+      codec => {{ LOGSTASH_INPUT_CODEC }}
+  }
+  tcp {
+      port => {{ LOGSTASH_INPUT_PORT }}
+      codec => {{ LOGSTASH_INPUT_CODEC }}
+  }
+}
+
+filter {
+    json_encode {
+        source => "docker"
+        target => "dimensions"
+    }
+    mutate {
+        remove_field => ["docker"]
+    }
+}
+
+output {
+  stdout { codec => rubydebug }
```

Comment:
I think that this basically is bad :D, but could be useful for debugging, so let's create a ```ENV DEBUG=False``` and only wrap the stoud with it.

Reply:
Done